com.retrodev.blastem.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<!-- Copyright 2019 Bastien Nocera -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.retrodev.blastem</id>
+  <launchable type="desktop-id">com.retrodev.blastem.desktop</launchable>
+  <name>Blastem</name>
+  <summary>A Sega Mega Drive/Genesis, and Master System emulator</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <description>
+    <p>
+      BlastEm is an open source, highly accurate emulator for the Genesis/Megadrive
+      that runs on modest hardware. It also supports playing back Sega Master System games.
+    </p>
+    <p>
+      Blastem has support for savestates, Mega/Sega Mouse, XBAND and Saturn keyboard emulation.
+    </p>
+  </description>
+  <url type="homepage">https://www.retrodev.com/blastem/</url>
+  <screenshots>
+    <screenshot type="default" width="640" height="517">https://github.com/flathub/com.retrodev.blastem/raw/master/screenshots/blastem-1.png</screenshot>
```

Comment:
@hadess well, they are not hosted anywhere. The URL points to a repository that doesn't exist yet. I would suggest using your current fork for the screenshots and update those with the new URL once the repo is created.